themes/default/theme/postcss.config.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Configuration file for PostCSS. This is only used as part of the Hugo pipeline
+ * build process for our Sass files, since we run the resulting CSS through PostCSS
+ * at the end to do more transformations.
+ */
+
+module.exports = {
+    plugins: [
+        // TailwindCSS
+        require("tailwindcss")("./tailwind.config.js"),
+
+        // Apply vendor prefixes for CSS features that aren't
```

Comment:
Should we just delete all this commented stuff?

Reply:
sure, we can delete it. I honestly pulled it from hugo and wasn't sure if there was a reason we were leaving it in there like for future reference or something.